game-app/game-core/src/test/java/games/strategy/engine/data/AllianceTrackerTest.java

```diff
@@ -2,13 +2,13 @@
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
This is causing one of several failed checks for the PR
```
> Task :game-app:domain-data:check

[ant:checkstyle] [WARN] /home/runner/work/triplea/triplea/game-app/game-core/src/test/java/games/strategy/engine/data/AllianceTrackerTest.java:5:47: Using the '.*' form of import should be avoided - org.junit.jupiter.api.Assertions.*. [AvoidStarImport]
```
Please make sure all checks are passed and if not provide additional commits to correct them.

Reply:
Yeah, I didn't notice this change locally when pushing up and assumed when I ran spotlessCheck it would fix things like this. I probably need to change an InetlliJ setting so it doesn't auto import *'s